depends/patches/qt/fix_android_get_drawable.patch

```diff
@@ -0,0 +1,20 @@
+--- old/qtbase/src/android/jar/src/org/qtproject/qt5/android/QtActivityDelegate.java
++++ new/qtbase/src/android/jar/src/org/qtproject/qt5/android/QtActivityDelegate.java
```

Comment:
```suggestion
Android 13: Fix warnings on starting an application

On Android 13, currently in beta, android triggers a warning because it
is using a deprecated getDrawable function.
The patch changes it to use the non-deprecated alternative as suggested
in the warning.

Fixes: QTBUG-103568

Upstream commits:
 - Qt 5.15.11 backport: unavailable
 - Qt 6.5: e07fea6fb405c6e9affe6e377ad99a98a740c5be


--- old/qtbase/src/android/jar/src/org/qtproject/qt5/android/QtActivityDelegate.java
+++ new/qtbase/src/android/jar/src/org/qtproject/qt5/android/QtActivityDelegate.java
```